app/views/languages/index.html.erb

```diff
@@ -1,38 +1,37 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts('The Archive supports these languages') %></h2>
+<h2 class="heading"><%= t(".description") %></h2>
```

Comment:
Can we call this `page_heading`? That way if we ever have to add a description to the page (i.e., a paragraph under the heading), we don't need to rename. It's also not a bad idea to have a consistent naming scheme for bits that appear on all pages, and it looks like that one is used pretty frequently.